src/yacs-web/src/pages/Admin.vue

```diff
@@ -15,21 +15,32 @@
         <input type="Submit" class="btn btn-success btn-sm" value="Submit" />
       </form>
     </section>
+    <section id="select-data">
```

Comment:
select-semester, data is ambiguous

Reply:
The idea is that there will be multiple data items to select, not just the default semester.